int/tests/conftest.py

```diff
@@ -168,3 +169,55 @@ async def http_endpoint(agent: BaseAgent):
     with suppress(asyncio.CancelledError):
         await server_task
     await agent.cleanup()
+
+
+@pytest.fixture
+async def make_did():
+    """DID factory fixture"""
+
+    def _make_did():
+        did = create_did.asyncio(client=backchannel)
+        return did
+
+    yield _make_did()
+    # TODO create DID deletion method
+
+
+@pytest.fixture
+async def make_endorser_did(make_did):
+    """Endorser DID factory fixture"""
+
+    def _make_endorser_did():
+        did = make_did()
+        print("Publishing DID through https://selfserve.indiciotech.io")
```

Comment:
We should probably remove explicit prints but getting this same detail through a Logger is probably a good idea.